src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -78,34 +79,34 @@
 (defn token-value-drawer
   [token watch-only? entry-point]
-  (let [token-symbol         (:token token)
-        token-data           (rf/sub [:wallet/token-by-symbol-from-first-available-account-with-balance
-                                      token-symbol])
-        selected-account     (rf/sub [:wallet/current-viewing-account-address])
-        token-owners         (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
-        testnet-mode?        (rf/sub [:profile/test-networks-enabled?])
-        account-owns-token?  (rf/sub [:wallet/current-account-owns-token token-symbol])
-        network-details      (rf/sub [:wallet/network-details])
-        receive-token-symbol (if (= token-symbol "SNT") "ETH" "SNT")
-        token-owned?         (if selected-account account-owns-token? (seq token-owners))
-        asset-to-receive     (rf/sub [:wallet/token-by-symbol receive-token-symbol])
-        unique-owner?        (= (count token-owners) 1)
-        params               (cond-> {:start-flow?   true
-                                      :owners        token-owners
-                                      :testnet-mode? testnet-mode?}
-                               selected-account
-                               (assoc :token        token-data
-                                      :stack-id     :screen/wallet.accounts
-                                      :has-balance? (-> (get-in token [:values :fiat-unformatted-value])
-                                                        money/above-zero?))
-                               (and (not selected-account) unique-owner?)
-                               (assoc :token-symbol token-symbol
-                                      :token        token-data
-                                      :stack-id     :wallet-stack)
+  (let [token-symbol        (:token token)
+        token-data          (rf/sub [:wallet/token-by-symbol-from-first-available-account-with-balance
+                                     token-symbol])
+        selected-account    (rf/sub [:wallet/current-viewing-account-address])
+        token-owners        (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
+        testnet-mode?       (rf/sub [:profile/test-networks-enabled?])
+        account-owns-token? (rf/sub [:wallet/current-account-owns-token token-symbol])
+        network-details     (rf/sub [:wallet/network-details])
+        token-owned?        (if selected-account account-owns-token? (seq token-owners))
+        asset-to-receive    (rf/sub [:wallet/token-by-symbol-from-first-available-account-with-balance
+                                     (swap-utils/default-asset-to-receive token-symbol)])
+        unique-owner?       (= (count token-owners) 1)
+        params              (cond-> {:start-flow?   true
+                                     :owners        token-owners
+                                     :testnet-mode? testnet-mode?}
+                              selected-account
```

Comment:
what `selected-account` means in this context? If it used for condition, it has some additional meaning in this code. Lets rename `selected-account` to reflect this meaning. wdyt?

Reply:
It's not part of this PR changes, here i just removed `receive-token-symbol`.